Classes/Service/OAuthService.php

```diff
@@ -67,22 +74,8 @@ public function setSettings(array $settings): self
      */
     public function getAuthorizationUrl(?ServerRequestInterface $request, array $options = []): string
     {
-        if (!empty($this->settings['oidcAuthorizeLanguageParameter'])) {
-            $languageOption = $this->settings['oidcAuthorizeLanguageParameter'];
-            if (!empty($languageOption)) {
-                $language = 'en';
-                if ($request) {
-                    /** @var SiteLanguage $siteLanguage */
-                    $siteLanguage = $request->getAttribute('language', $request->getAttribute('site')->getDefaultLanguage());
-                    // fallback for TYPO3 v11
-                    $language = is_object($siteLanguage->getLocale())
-                        ? $siteLanguage->getLocale()->getLanguageCode()
-                        : $siteLanguage->getTwoLetterIsoCode();
-                }
-                $options[$languageOption] = $language;
-            }
-        }
-
+        $event = $this->eventDispatcher->dispatch(new GetAuthorizationUrlEvent($request, $this->settings, $options));
+        $options = $event->getOptions();
```

Comment:
This getter does not exist

Reply:
Oh yes, thank you. Good catch. This must have happened during the rebase